ql/src/test/queries/clientpositive/perf/cbo_query74.q

```diff
@@ -1,3 +1,4 @@
+--! qt:disabled:HIVE-23963
```

Comment:
this will disable this test in all drivers - in case we will only use the new driver it will be clearer...

Reply:
The issue has been fixed but anyways we are using only the new driver now.